frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -17,21 +17,32 @@ const PVCStatus = ({pvc}) => {
 };
 
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-sm-4 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.volumeName">Persistent Volume</ColHead>
+  <ColHead {...props} className="col-sm-3 hidden-xs" sortField="spec.resources.requests.storage">Requested</ColHead>
```

Comment:
This is too much to fit in the table at the `sm` breakpoint. We need to be more aggressive about dropping columns and only show everything at `md` or `lg` widths.

Reply:
ack